frontend/src/metabase/query_builder/components/view/sidebars/QuestionDetailsSidebar.jsx

```diff
@@ -3,15 +3,34 @@ import PropTypes from "prop-types";
 
 import SidebarContent from "metabase/query_builder/components/SidebarContent";
 import QuestionActionButtons from "metabase/questions/components/QuestionActionButtons";
+import { PLUGIN_MODERATION_COMPONENTS } from "metabase/plugins";
+
+const {
+  active: isPluginActive,
+  ModerationIssueActionMenu,
+} = PLUGIN_MODERATION_COMPONENTS;
 
 function QuestionDetailsSidebar({ question, onOpenModal }) {
   const canWrite = question && question.canWrite();
 
   return (
     <SidebarContent className="full-height px1">
-      <div>
-        <QuestionActionButtons canWrite={canWrite} onOpenModal={onOpenModal} />
-      </div>
+      {isPluginActive ? (
+        <div>
+          <QuestionActionButtons
+            canWrite={canWrite}
+            onOpenModal={onOpenModal}
+          />
+          <ModerationIssueActionMenu onAction={() => {}} />
+        </div>
+      ) : (
+        <div>
+          <QuestionActionButtons
+            canWrite={canWrite}
+            onOpenModal={onOpenModal}
+          />
+        </div>
+      )}
```

Comment:
Not a big deal, these two nodes seem to be very similar, should we maybe make it a bit more compact?


```jsx
<div>
 <QuestionActionButtons
    canWrite={canWrite}
    onOpenModal={onOpenModal}
 />
 {isPluginActive && <ModerationIssueActionMenu onAction={() => {}} />}
</div>
```

Reply:
This is going to change a lot over the next few PRs I'm doing, so I won't worry about being a little too redundant for now.